### ISSUE TITLE
[Bug]: Thumbnail links in playlists.db refer to the invidious instance used when adding the video, not to YT

### ISSUE THREAD

**Issue by TheodorSmall** [FreeTubeApp/FreeTube#5601]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Use invidious API (preferably disable "fallback to secondary API") and select *instance 1*
2. Create a playlist and add a video to it
3. Optionally change the invidious instance to *instance 2* and assume *instance 1* is no longer working
4. In playlists.db, you will find that the thumbnail links are still pointing to *instance 1*, and therefore they depend on an instance that is no longer selected by the user.

### Expected Behavior

The thumbnail links should point to YT, and dynamically the correct, current instance should be used to receive the thumbnail.

### Issue Labels

accessibility issue, content not loading, inconsistent behavior

### FreeTube Version

v0.21.3 Beta (pacman built from source)

### Operating System Version

Artix Linux

### Installation Method

.pacman

### Primary API used

Invidious API

### Last Known Working FreeTube Version (If Any)

unknown, probably none

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by TheodorSmall** [comment]
This also is a privacy issue, since the user may want to actively avoid using *instance 1*.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I cant seem to find the identifier for the thumbnail links that are listed in the playlists.db. Are you sure you are talking about the right file?

**Comment by TheodorSmall** [comment]
In fact, playlists.db is the right file, and although the video thumbnails should not be saved there, they are saved with the playlist, however only when adding multiple videos at once. The fix is quite easy because there is already a list of "undesired attributes" in the _addVideos_ function in _src/renderer/store/modules/playlists.js_.
There are also a few other attributes that are only saved when adding multiple videos. These are all additional attributes that are not yet contained in the list:
- videoThumbnails
- authorUrl
- liveNow
- index

This is not a privacy issue, since the links in question are never read.
